next.config.mjs

```diff
@@ -78,28 +83,35 @@ const sentrySettings = {
   org: 'nodejs-org',
   // Define the Sentry Project on our Sentry Organisation
   project: 'nodejs-org',
+  // Sentry DSN for the Node.js Website
+  dsn: SENTRY_DSN,
 };
 
 /** @type {import('@sentry/nextjs/types/config/types').UserSentryOptions} */
 const sentryConfig = {
   // Upload Next.js or third-party code in addition to our code
   widenClientFileUpload: true,
-  // Transpile the Sentry code too since we target older browsers in our .browserslistrc
-  transpileClientSDK: true,
```

Comment:
I don't know how it affects the bundle size, but it would be nice to transpile according to the our `.browserslistrc` 👀